### ISSUE TITLE
Arangoimport imports on _system when you try to create a new database

### ISSUE THREAD

**Issue by 38xela** [arangodb/arangodb#10183]
## My Environment

* __ArangoDB Version__:        3.5.0
* __Storage Engine__:             RocksDB
* __Deployment Mode__:        Single Server
* __Deployment Strategy__:   ArangoDB Starter
* __Configuration__:               
* __Infrastructure__:               AWS
* __Operating System__:        4.14.138-89.102.amzn1.x86_64
* __Total RAM in your machine__:        122 GiB
* __Disks in use__:        SSD
* __Used Package__:              SUSE or RedHat .rpm

## Component, Query & Data

__Affected feature__:
arangoimport

__Size of your Dataset on disk__:
80GB

## Steps to reproduce
`arangoimport --type jsonl --create-database --collection pages_uk --threads 16 --file /data/arango_dump/pages_uk.jsonl --create-collection --create-collection-type document --on-duplicate ignore --progress --server.endpoint=http+tcp://127.0.0.1:8529 --server.database crawler_db`

__Problem__:
```
Please specify a password:
Creating database 'crawler_db'
Connected to ArangoDB 'http+tcp://127.0.0.1:8529, database: '_system', username: 'root'
----------------------------------------
database:               _system
collection:             pages_uk
create:                 yes
create database:        yes
source filename:        /data/arango_dump/pages_uk.jsonl
file type:              jsonl
threads:                16
connect timeout:        5
request timeout:        1200
----------------------------------------
Starting JSON import...
```

__Expected result__:
```
Please specify a password:
Creating database 'crawler_db'
Connected to ArangoDB 'http+tcp://127.0.0.1:8529, version: 3.5.0, database: 'crawler_db', username: 'root'
----------------------------------------
database:               crawler_db
collection:             pages_uk
create:                 yes
create database:        yes
source filename:        /data/arango_dump/pages_uk.jsonl
file type:              jsonl
threads:                16
connect timeout:        5
request timeout:        1200
----------------------------------------
Starting JSON import...
```

**Comment by jsteemann** [comment]
As far as I can see, the import still imports into the correct database (`crawler_db` in this case), but the output of arangoimport is wrong, as it shows `_system` as the target database.

**Comment by jsteemann** [comment]
Can you confirm this is the case for you as well, i.e. that all data has been imported into the correct database, despite the wrong output of arangoimport?

**Comment by 38xela** [comment]
hi @jsteemann the data has been imported in the wrong database (`_system`), so the output is correct but not the arangoimport behavior.

**Comment by jsteemann** [comment]
@38xela : I can't reproduce the problem that the data is imported into the wrong database, neither if the target database already exists or not.
Is it possible that, for whatever reason, the `pages_uk` collection already existed in the `_system` database before this import run?

**Comment by jsteemann** [comment]
Will now create PRs that will fix the wrong output of arangoimport.

**Comment by jsteemann** [comment]
@38xela: The above PRs fix the output of arangoimport, concerning the wrong database name.
I haven't been able to reproduce the issue of the data getting imported into a wrong database, so I suspect the collection may have been present in the `_system` database before.
If you can reproduce the issue somehow and can rule out the target collection previously existed in the `_system` database, please let me know the exact steps for reproduction.
Thanks!

**Comment by maxkernbach** [comment]
Hi @38xela,

Version 3.5.3, which includes a fix for the arangoimport display issue, has been released.
As mentioned in the previous comment, this fix does not apply to the actual database where the data is getting imported into.

From the [Changelog](https://github.com/arangodb/arangodb/blob/660ff994dbfa994a18fe921481f591b878b51619/CHANGELOG#L142):

> * Fixed issue #10183: Arangoimport imports on _system when you try to
>   create a new database.
> 
>   This bugfix fixes the output of arangoimport, which could display a
>   wrong target database for the import if the option `--create-database`
>   was used.
> 

In case you can reproduce importing data with arangoimport into a wrong database, please share the steps on how to reproduce and we would be glad to further investigate this issue.

Please find the latest release in the [download section](https://www.arangodb.com/download-major/).
